### PR TITLE
Convert call list to be in pycg format

### PR DESCRIPTION
**Pull request by zl81**
In this pull request a script `pycg_convert.py` is created inside `inspect4py_pycg`, this script takes an input path, generates call list using `utils.py/call_list_file` or `utils.py/call_list_dir` and converts it into pycg format. I have also added a TODO comment indicating a potential join path issue in `cli.py`.

Please feel free to edit as you see fit.

### CODE REVIEW

File: inspect4py_pycg/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## Pycg format converter
+
+`pycg_convert.py` converts the call list generated by inspect4py to be in the format used by pycg.
+
+Usage:
+```
+python3 pycg_convert.py <input_path>
+```
+    
+where `<input_path>` could be the path to the repository directory or the path to
```
Comment (dgarijo) [inline inspect4py_pycg/README.md:10]: Could be or it is?

File: inspect4py/cli.py [hunk 2]
```diff
@@ -1249,7 +1249,7 @@ def main(input_path, fig, output_dir, ignore_dir_pattern, ignore_file_pattern, r
                 if ".py" in f and not f.endswith(".pyc"):
                     try:
                         path = os.path.join(subdir, f)
-                        out_dir = output_dir + "/" + os.path.basename(subdir)
+                        out_dir = output_dir + "/" + os.path.basename(subdir) # TODO: This could create path like /parent//child
```
Comment (dgarijo) [inline inspect4py/cli.py:1252]: If you add a TO DO, please open an issue and point to this comment

**Comment by dgarijo** [comment]
@zl81,
thanks. I have reviewed this PR. However, I don't think this is nicely integrated into the inspect4py framework.
If you want to have a converter, that's fine, but it should be integrated with a command, in this case as an auxiliary file. For example something like :
`inspect4py -cl --format pycg` 

@rosafilgueira what do you think?

**Comment by zl81** [comment]
> @zl81, thanks. I have reviewed this PR. However, I don't think this is nicely integrated into the inspect4py framework. If you want to have a converter, that's fine, but it should be integrated with a command, in this case as an auxiliary file. For example something like : `inspect4py -cl --format pycg`
> 
> @rosafilgueira what do you think?

Thanks for your feedback, my current thought is that converting to pycg format is just a side product for inspect4py to compare its performance with pycg. As it is not a core functionality of inspect4py, instead of integrating it, I added it as a script.

File: inspect4py_pycg/README.md (hunk 1 above)
Comment (zl81) [reply inspect4py_pycg/README.md:10]: Thanks, I will rephrase it.

**Comment by dgarijo** [comment]
Ok, then I misinterpreted the functionality. It's ok having a separate script. Please do open an issue with the pending to do :)

File: inspect4py/cli.py (hunk 2 above)
Comment (zl81) [reply inspect4py/cli.py:1252]: The issue has been created: https://github.com/SoftwareUnderstanding/inspect4py/issues/361#issue-1353004257.

**Comment by rosafilgueira** [comment]
Hi @dgarijo, yes. This functionality is outside inspect4py. Basically it takes the call_list generated by inspect4py and "transformed" it into the call_list format used by pycg. This will enable us to compare both call_lists or call_graphs by both tools in the future.

**Pull request merged**